warehouse/packaging/models.py

```diff
@@ -49,7 +49,6 @@
 from warehouse.accounts.models import User
 from warehouse.classifiers.models import Classifier
 from warehouse.events.models import HasEvents
-from warehouse.integrations.vulnerabilities.models import VulnerabilityRecord
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏was this to avoid a circular import, or for some other reason? When we come to annotate this line, we'll likely need to bring the import back.

Reply:
Yes, this was to avoid a circular import, wasn't really clear to me how we'd handle circular imports with types + two way relationships.

Reply:
Hmmm I don't know the answer to that yet - something to ruminate on.

I wonder if there's a need to express the relationship on both sides if we use back_populates - we should dig into that at some point, but definitely not a blocker.